frontend/src/tokens.css

```
@@ -53,8 +52,8 @@
   --space-6: 16px;
   --space-7: 20px;
   --space-8: 24px;
-  --font-display: ;
-  --font-sans: ;
+  --font-display: "Space Grotesk", Inter, system-ui, sans-serif;
```

Stylelint (17.5.0)

[error] Expected "Inter" to be "inter" (value-keyword-case)

(value-keyword-case)